printer_mods/Zen3D/V0_Skirt/README.md

```diff
@@ -0,0 +1,46 @@
+# Voron0 dual color skirt set (SSR, PowerSwitch, Nameplate)
+
+To match the skirt with the [Midpanel with Hex Pattern](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Zen3D/V0_MidPanel_HexPattern)  by sterminatore's idea
+
+![img3](./IMG/img1.jpg)
+
+![img3](./IMG/img2.jpg)
+
+## print settings:
+
+- layer height: 0.2mm
+- color change at 0.6mm / 2.2mm
+
+## Skirt.stl, Skirt_mirror.stl
+
+Left and right skirt part with 40 degree chamfers for color change printing. There is an extra hole for tightening the middle M3 screw.
+
+![img5](./IMG/img5.jpg)
+
+## Skirt_EmptyNameplate.stl
+
+Empty nameplate for serial number. For example, with Tinkercad you can easily put the number on it.
```

Comment:
Can you add a tutorial (maybe at the end of this README or as a separate file and link to it) on how to add the number using Tinkercad, assuming the user has never used Tinkercad before.

Reply:
I made a clumsy video about using tinkercad. :-)